### ISSUE TITLE
Incompatibility with react 15.4.0

### ISSUE THREAD

**Issue by ldabiralai** [insin/react-maskedinput#76]
React 15.4.0 has been released which removes a lot of the internals, specifically around `ReactInputSelection` which causes `3.3.0` to break. Saw this had been remedied in the latest commits though.

It would be good to get this released soon as it's likely a lot of people will be upgrading in the next few days.

Thanks 👍

**Comment by yocontra** [comment]
```
ERROR in ./~/react-maskedinput/lib/index.js
Module not found: Error: Cannot resolve module 'react/lib/ReactInputSelection' in /Users/contra/Projects/staeco/node_modules/react-maskedinput/lib
 @ ./~/react-maskedinput/lib/index.js 9:15-55
```

**Comment by rfviolato** [comment]
:+1:

I was about to open the same issue!

**Comment by pixelfuture** [comment]
Getting the same error when upgrading to react v15.4

**Comment by imchuck** [comment]
Hope you fix this issue soon 👍

**Comment by benitocas88** [comment]
We have the same problem!!! Help

**Comment by radoi90** [comment]
It was fixed two days ago, the commit is there but no release. What gives?

**Comment by sarcastron** [comment]
Not sure if it helps anyone else. I was having the same issue and got around it by changing dependencies back to `15.03`.

```
{
    "react": "~15.3.1",
    "react-dom": "~15.3.1"
}
```

Basically switched the `^` to `~`.

I noticed that the commit looks like it fixes it, but it didn't seem to work if I forced npm to pull from git repo instead of NPM.

**Comment by markbrouch** [comment]
I'm seeing the issue in the es6 generated file in `/es6/index.js` and `/lib/index.js` on L9:

``` js
var _require = require('react/lib/ReactInputSelection');
```

**Comment by markbrouch** [comment]
@radoi90 Looks like it just hasn't been published since the last change: https://www.npmjs.com/package/react-maskedinput

edit: it has been published since the last change, but it wasn't built first. @iamdustan will have to build and republish for this to be fixed.

**Comment by prabhath** [comment]
We are facing the same issue

**Comment by rfviolato** [comment]
Yeah, the changes are not published :/

Well i guess i will copy the source to my project and use it as a custom component until they fix this issue.

**Comment by imchuck** [comment]
Any update on this? Where are you planning to republish to npm this fix?

**Comment by tonygentilcore** [comment]
Anything we can do to help get the fix released? Really excited to upgrade to 15.4.

**Comment by nguyenhose** [comment]
same problem here, it's awful when happened on deploy day.

**Comment by KittyGiraudel** [comment]
Yo! Thanks for the super quick fix. Any chance we could have a published build soon? :)

**Comment by hunterfortuin** [comment]
+1 on this! Would love to use the NPM package instead of copy/pasta code in my project.

**Comment by iamdustan** [comment]
Heyo. Apologies for the delay. I was on holiday with my family. Published react-maskedinput@3.3.1
